frameworks/FSharp/zebra/zebra-simple.dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0 AS build
 WORKDIR /app
 COPY src/App .
 RUN dotnet publish -c Release -o out
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:2.2 AS runtime
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.0 AS runtime
 ENV ASPNETCORE_URLS http://+:8080
 ENV COMPlus_ReadyToRun 0
```

Comment:
This env var was removed from the other dotnet core 3.0 docker files

Reply:
Yeah, tiered compilation in 3.0 means this is no longer needed

Reply:
You mean that complus thing?

Reply:
Yea, remove that one line
```
ENV COMPlus_ReadyToRun 0
```

Reply:
thanks. done